main/main.cpp

```diff
@@ -680,6 +680,8 @@ int Main::test_entrypoint(int argc, char *argv[], bool &tests_need_run) {
  */
 
 Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_phase) {
+	Thread::make_main_thread();
```

Comment:
`Main::test_setup()` probably should do the same.

Reply:
done, thanks!